Header header

float32 pressure    # in pascals
float32 temperature # deg. celcius
float32 baro_zpos   #

float32[3] mag      # magnetic field strength
					# x, y, z
